code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -232,6 +233,7 @@
 	..()
 	M.adjustToxLoss(1, 0)
 	M.adjustOrganLoss(ORGAN_SLOT_BRAIN, pick(0.5, 0.6, 0.7, 0.8, 0.9, 1))
+	M.adjust_bodytemperature(25 * TEMPERATURE_DAMAGE_COEFFICIENT, BODYTEMP_NORMAL, INFINITY) //meth OD makes you VERY warm
```

Comment:
```suggestion
	M.adjust_bodytemperature(2, BODYTEMP_NORMAL, INFINITY) //meth OD makes you VERY warm
```

Reply:
One of Meth's OD symptoms IRL is a severely elevated body-temp, to the point that doctors may need to place you on bags of ice just to keep your body from cooking itself to death